src/workerd/api/queue.c++

```diff
@@ -179,21 +190,30 @@ kj::Promise<void> WorkerQueue::send(jsg::Lock& js,
 
   JSG_REQUIRE(!body.isUndefined(), TypeError, "Message body cannot be undefined");
 
+  auto headers = kj::HttpHeaders(context.getHeaderTable());
+  headers.set(kj::HttpHeaderId::CONTENT_TYPE, MimeType::OCTET_STREAM.toString());
+
   kj::Maybe<kj::StringPtr> contentType;
   KJ_IF_SOME(opts, options) {
     KJ_IF_SOME(type, opts.contentType) {
       contentType = validateContentType(type);
+      headers.add(HDR_MSG_FORMAT, type);
```

Comment:
We should continue using `contentType` for this (as before) so that we always send the standardized lower-case version rather than whatever the user gave us.

In this case you may need to use `KJ_ASSERT_NONNULL(contentType)` in order to make it compile since the `contentType` variable is a `kj::Maybe`. Or just do something like:

```
auto validatedType = validateContentType(type);
contentType = validatedContentType;
headers.add(HDR_MSG_FORMAT, validatedContentType);
```

Either is fine.